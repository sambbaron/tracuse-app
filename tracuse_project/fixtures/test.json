{
  datumtype: [
    {
      "active": true,
      "sort": 1010,
      "short_definition": null,
      "entity_name": "Category",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 1,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 1020,
      "short_definition": null,
      "entity_name": "Project",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 2,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 1030,
      "short_definition": null,
      "entity_name": "Context",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 3,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 1040,
      "short_definition": null,
      "entity_name": "Person",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 4,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 1050,
      "short_definition": null,
      "entity_name": "Entity",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 5,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 1060,
      "short_definition": null,
      "entity_name": "Location",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 6,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 1070,
      "short_definition": null,
      "entity_name": "Contact",
      "repr_expression": "name",
      "datum_group": 1,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 7,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 2010,
      "short_definition": null,
      "entity_name": "Action",
      "repr_expression": "name",
      "datum_group": 2,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 8,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 2020,
      "short_definition": null,
      "entity_name": "Principle",
      "repr_expression": "name",
      "datum_group": 2,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 9,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 2030,
      "short_definition": null,
      "entity_name": "Journal",
      "repr_expression": "name",
      "datum_group": 2,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 10,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 3010,
      "short_definition": null,
      "entity_name": "Schedule",
      "repr_expression": "name",
      "datum_group": 3,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 11,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 3020,
      "short_definition": null,
      "entity_name": "Event",
      "repr_expression": "name",
      "datum_group": 3,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 12,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 3030,
      "short_definition": null,
      "entity_name": "Reminder",
      "repr_expression": "name",
      "datum_group": 3,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 13,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 4010,
      "short_definition": null,
      "entity_name": "Note",
      "repr_expression": "name",
      "datum_group": 4,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 14,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 4020,
      "short_definition": null,
      "entity_name": "File",
      "repr_expression": "name",
      "datum_group": 4,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 15,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 4030,
      "short_definition": null,
      "entity_name": "Directory",
      "repr_expression": "name",
      "datum_group": 4,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 16,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 4040,
      "short_definition": null,
      "entity_name": "Webpage",
      "repr_expression": "name",
      "datum_group": 4,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 17,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 4050,
      "short_definition": null,
      "entity_name": "Message",
      "repr_expression": "name",
      "datum_group": 4,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 18,
      "schema_name": ""
    },
    {
      "active": true,
      "sort": 4060,
      "short_definition": null,
      "entity_name": "Post",
      "repr_expression": "name",
      "datum_group": 4,
      "readable_name": "",
      "long_definition": null,
      "plural_name": "",
      "datum_type_id": 19,
      "schema_name": ""
    }
  ]
}
