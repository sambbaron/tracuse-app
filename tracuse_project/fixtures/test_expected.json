{
  "datumtype": [
    {
      "readable_name": "",
      "datum_type_id": 1,
      "sort": 1010,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Category",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 2,
      "sort": 1020,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Project",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 3,
      "sort": 1030,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Context",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 4,
      "sort": 1040,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Person",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 5,
      "sort": 1050,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Entity",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 6,
      "sort": 1060,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Location",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 7,
      "sort": 1070,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Contact",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 1
    },
    {
      "readable_name": "",
      "datum_type_id": 8,
      "sort": 2010,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Action",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 2
    },
    {
      "readable_name": "",
      "datum_type_id": 9,
      "sort": 2020,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Principle",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 2
    },
    {
      "readable_name": "",
      "datum_type_id": 10,
      "sort": 2030,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Journal",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 2
    },
    {
      "readable_name": "",
      "datum_type_id": 11,
      "sort": 3010,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Schedule",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 3
    },
    {
      "readable_name": "",
      "datum_type_id": 12,
      "sort": 3020,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Event",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 3
    },
    {
      "readable_name": "",
      "datum_type_id": 13,
      "sort": 3030,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Reminder",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 3
    },
    {
      "readable_name": "",
      "datum_type_id": 14,
      "sort": 4010,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Note",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 4
    },
    {
      "readable_name": "",
      "datum_type_id": 15,
      "sort": 4020,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "File",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 4
    },
    {
      "readable_name": "",
      "datum_type_id": 16,
      "sort": 4030,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Directory",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 4
    },
    {
      "readable_name": "",
      "datum_type_id": 17,
      "sort": 4040,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Webpage",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 4
    },
    {
      "readable_name": "",
      "datum_type_id": 18,
      "sort": 4050,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Message",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 4
    },
    {
      "readable_name": "",
      "datum_type_id": 19,
      "sort": 4060,
      "long_definition": null,
      "repr_expression": "name",
      "short_definition": null,
      "active": true,
      "entity_name": "Post",
      "plural_name": "",
      "schema_name": "",
      "datum_group": 4
    }
  ]
}
